Atreya/clinics/urls.py

```diff
@@ -4,7 +4,8 @@
 
 urlpatterns = [
     path('clinics/total_clinics', views.total_clinics, name='total-clinics'),
-    path('clinics/<int:clinic_id>',views.clinics),  
+    path('clinics/<int:clinic_id>',views.clinics),
+    path('clinics/get_nearby_clinics/<id>', nearby_clinics),  
```

Comment:
looking at your views class, this doesn't need to be a separate URL endpoint, it can be the same function differentiated via input variables